src/line/i-chartBuilder.test.js

```diff
@@ -0,0 +1,400 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const fs = require("fs")
+const domTesting = require('@testing-library/dom')
+require('@testing-library/jest-dom')
+const userEvent = require("@testing-library/user-event").default
+
+//Spy on
+const window = require("./window")
```

Comment:
Wait, isn't the `window` object supposed to be created by `jsdom`? I don't think we need to `require()` it directly.